tests/moduletool/test_install.py

```diff
@@ -1263,32 +1262,16 @@ def test_module_install_logging(local_module_package_index: str, snippetcompiler
         )
 
 
-@pytest.mark.slowtest
-def test_real_time_logging(caplog):
```

Comment:
The goal of this test is to verify `run_command_and_stream_output` logs in real time. We should keep that test. The only thing we cannot keep is the command it executes, because it requires `shell=True`, which is removed by this PR. But there are ways to work around this by changing the command in the following way for example:

```
cmd: list[str] = ["sh", "-c", "echo one && sleep 1 && echo two"]
```